### ISSUE TITLE
Rule S2068: filter empty strings

### ISSUE THREAD

**Issue by pierre-loup-tristant-sonarsource** [SonarSource/sonar-dotnet#2869]
[RSPEC-2068](https://jira.sonarsource.com/browse/RSPEC-2068)
The rule should avoid raising issues when the litteral string is empty:

```csharp
string password = ""; // Compliant
string pwd = ""; // Compliant
```

**Comment by costin-zaharia-sonarsource** [comment]
This is no longer reproducible on C# side and covered by the test cases: https://github.com/SonarSource/sonar-dotnet/blob/master/sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotHardcodeCredentials_DefaultValues.cs#L33
